packages/gutenpride-template/src/src/index.js.mustache

```diff
@@ -0,0 +1,98 @@
+/**
+ * Registers a new block provided a unique name and an object defining its behavior.
+ *
+ * @see https://developer.wordpress.org/block-editor/developers/block-api/#registering-a-block
+ */
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Retrieves the translation of text.
+ *
+ * @see https://developer.wordpress.org/block-editor/packages/packages-i18n/
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Sass files
+ */
+import './style.scss';
+import './editor.scss';
+
+/**
+ * Internal dependencies
+ */
+import Edit from './edit';
+import save from './save';
+
+/**
+ * Every block starts by registering a new block type definition.
+ *
+ * @see https://developer.wordpress.org/block-editor/developers/block-api/#registering-a-block
+ */
+registerBlockType('{{namespace}}/{{slug}}', {
+    /**
+     * This is the display title for your block, which can be translated with `i18n` functions.
+     * The block inserter will show this name.
+     */
+    title: __('{{title}}', '{{textdomain}}'),
+
+    {{#description}}
+    /**
+     * This is a short description for your block, can be translated with `i18n` functions.
+     * It will be shown in the Block Tab in the Settings Sidebar.
+     */
+    description: __(
+        '{{description}}',
+        '{{textdomain}}'
+    ),
+
+    {{/description}}
+
+    /**
+     * Blocks are grouped into categories to help users browse and discover them.
+     * The categories provided by core are `common`, `embed`, `formatting`, `layout` and `widgets`.
+     */
+    category: '{{category}}',
+
+	{{#dashicon}}
+    /**
+     * An icon property should be specified to make it easier to identify a block.
+     * These can be any of WordPress’ Dashicons, or a custom svg element.
+     */
+    icon: '{{dashicon}}',
+
+    {{/dashicon}}
+
+    /**
+    * Attributes are the way a block stores data, they define how a block is parsed to extract data from the saved content.
+    * When the block loads it will look at the saved content for the block, look for the div tag, take the text portion, and store the content in an attributes.message variable.
+    *
+    * @see https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/block-api/block-attributes.md
```

Comment:
Another place where there is an alternative URL
```suggestion
    * @see https://developer.wordpress.org/block-editor/developers/block-api/block-attributes/
```

Reply:
Done 👍